res/games/gameitem.css

```diff
@@ -0,0 +1,138 @@
+{%- macro playermacro(player,align) %}
+        {% if player.login is not defined %}
+            {% set player_login = player %}
+        {% else %}
+            {% set player_login = player.login %}
+        {% endif %}
+        
+        {% if player.countrypath is not defined %}
+            {% set player_country = "" %}
+        {% else %}
+            {% set player_country = player.countrypath %}
+            {% set player_global_rating = (player.global_rating[0]-(player.global_rating[1]*3))|int %}
+        {% endif %}
+
+        {% if teams[1] is not defined %}
+            {% set alignment = 'left' %}
+            {% if "-" not in player.login %}
+	            {% set width = 0 %}
+	        {% else %}
+	            {% set width = 130 %}
+	        {% endif %}
+        {% else %}
+            {% set alignment = 'right' %}
+	        {% set width = 130 %}
+        {% endif %}
+        <tr>
+        {% if align == "left" -%}
+            <td align={{ alignment }} width='{{ width }}'>
+            {{ player_login }}
+            </td>
+            <td>
+            <img src='{{ player_country }}'>
+            </td>
+            <td align={{ alignment }} valign='middle' width='10'>
+            {% if player_global_rating is defined %}
+                ({{ player_global_rating }})
+            {% else %}
+                (???)
+            {% endif %}
+            </td>
+        {% else %}
+            <td align='left' valign='middle' width='10'>
+            {% if player_global_rating is defined %}
+                ({{ player_global_rating }})
+            {% else %}
+                (???)
+            {% endif %}
+            </td>
+            <td>
+            <img src='{{ player_country }}'>
+            </td>
+            <td align='left' width={{ width }}>
+            {{ player_login }}
+            </td>
+        {% endif %}
+         </tr>
+{% endmacro -%}
+
+
+{%- macro versus_string(fontsize) %}
+    <td align='middle' valign='middle' height='100%'>
+    <font color ='black' size={{ fontsize }}>VS</font>
+    </td>
+{% endmacro -%}
+
+
+{%- macro modtip(mods) %}
+    <b><br/>With:</b>
+    {% for mod in mods %}
+        <br/> {{ mod }}
+    {% endfor %}
+{% endmacro -%}
+
+
+{%- macro obstip(observers) %}
+    <b>Observers:</b>
+    {% for observer in observers %}
+        <br/>{{ observer.login }} <img src='{{ observer.countrypath }}'> ({{ (observer.global_rating[0]-(observer.global_rating[1]*3))|int }})
+    {% endfor %}
+{% endmacro -%}
+
+
+{%- macro formatting(teams, mods, observers) %}
```

Comment:
What do we format by `formatting`?? Teams, mods, observers? Bad name for macro, that formats game info.

Reply:
again, i took this from the original code, where the final function was called "format" - but ill change it to something like "TooltipFormatting"

Reply:
rename it in better way, don't just copy code, make it better! :-)